File: kotlin/PlanesAndroid/app/src/test/java/ComputerLogicTest.java
import com.google.common.truth.Truth;
import com.planes.common.ComputerLogic;

import org.junit.Test;
import static com.google.common.truth.Truth.assertThat;

public class ComputerLogicTest {

    @Test
    public void ComputerLogic_MapIndexToPlane() {
        ComputerLogic cl = new ComputerLogic(10, 10, 3);
        assertThat(cl.mapPlaneToIndex(cl.mapIndexToPlane(50))).isEqualTo(50);
    }
}

File: kotlin/PlanesAndroid/app/src/test/java/ComputerLogicTest.kt
import com.google.common.truth.Truth
import com.planes.single_player_engine.ComputerLogic
import org.junit.Test

class ComputerLogicTest {
    @Test
    fun ComputerLogic_MapIndexToPlane() {
        val cl = ComputerLogic(10, 10, 3)
        Truth.assertThat(cl.mapPlaneToIndex(cl.mapIndexToPlane(50))).isEqualTo(50)
    }
}